src/Components/Artwork/Details/BidTimerLine.tsx

```diff
@@ -13,22 +15,30 @@ interface BidTimerLineProps {
 export const BidTimerLine: React.FC<BidTimerLineProps> = ({ artwork }) => {
   const data = useFragment(bidTimerLineFragment, artwork)
 
-  const { collectorSignals } = data
+  const { collectorSignals, saleArtwork } = data
   const { lotClosesAt, registrationEndsAt, onlineBiddingExtended } =
     collectorSignals?.auction ?? {}
-  const { time } = useTimer(lotClosesAt ?? "")
+
+  const [updatedLotClosesAt, setUpdatedLotClosesAt] = useState(lotClosesAt)
+
+  useAuctionWebsocket({
+    lotID: saleArtwork?.lotID!,
+    onChange: ({ extended_bidding_end_at }) => {
+      setUpdatedLotClosesAt(extended_bidding_end_at)
```

Comment:
One other nuance @starsirius , in order for the display to flip to have the 'Extended' label properly, if you load the page before the lot is extended and then the websocket happens - you may need another local state `isExtended: boolean`, that gets set to true when a websocket event comes in, so you can use that to render 'Extended' label.

Like https://github.com/artsy/force/blob/7b1e4283cee741c6768d5c72e89582a460845a61/src/Components/Artwork/Details/Details.tsx#L466